bin/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :name
+  attr_reader :publish_date
+
+  def initialize(name, publish_date, on_spotify)
+    super(publish_date)
+    @name = name
+    @publised_date = publised_date
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    @publish_date < 10.year.ago && @on_spotify
```

Comment:
I tested this code and doesn't work outside `Rails` environment.

Look at this alternative:
```Ruby
  def can_be_archived?
    @publish_date < n_years_ago(10) && @on_spotify
  end

  def n_years_ago(years)
    year = Time.now.year - years
    month = Time.now.month
    day = Time.now.day

    Time.new(year, month, day)
  end
```

Reply:
Thank you very much, I have done the corrections